templates/overrides/search/block--facets-summary-block.html.twig

```diff
@@ -14,7 +14,14 @@
     {{ title_prefix }}
     {% if label %}
       <div class="col-md-6 col-lg-8 align-self-center">
-        <h4 class="mb-0"><span class="text-capitalize">{{ label }} </span> {{ results_count }}</h4>
+        {% set _title %}
+          {{ label|render|striptags }} ({{ results_count }})
+        {% endset %}
```

Comment:
For me this results in duplicate '(': "Search Results ((19))".
This is because the '(' is already in `facets-summary-count.html.twig`.

Also the `|render|striptags` is pointless here because unlike for list page, `label` is a string, not a field render element.

Let's move this change to a follow-up issue, following our conversation last week.